configuration.go

```diff
@@ -0,0 +1,85 @@
+package caddy
+
+import (
+	"github.com/darkweak/souin/configurationtypes"
+	"go.uber.org/zap"
+)
+
+// DefaultCache the struct
+type DefaultCache struct {
+	Distributed bool
+	Headers     []string
+	Olric       configurationtypes.CacheProvider
+	Regex       configurationtypes.Regex
+	TTL         string
+}
+
+// GetDistributed returns if it uses Olric or not as provider
+func (d *DefaultCache) GetDistributed() bool {
```

Comment:
All these getters are useless and not idiomatic in Go, you can access to the field directly.

Reply:
That's for the configuration interface, so it's mandatory.